README.md

```diff
@@ -71,7 +71,9 @@ Here's a comparison, the first graph is rescheduling before enabling it, and the
 
 ## Easy Days
 
-You can choose any day or days from Monday to Sunday to take off. Once enabled, the Helper will try to avoid these days when rescheduling. Note: Easy Days only works for review cards. Due to technical limitations, FSRS doesn't modify the interval and due date of (re)learning cards. And it also doesn't reschedule cards whose interval is less than 3 days to respect the desired retention.
+You can choose any day or days from Monday to Sunday to take off. Once enabled, the Helper will try to avoid these days when rescheduling. In "Reduced" days, you only have ~50% of your usual reviews. In "Minimum" days, there is ~0% reviews.
```

Comment:
You can select any day or days from Monday through Sunday to take off. Once enabled, the Helper add-on will try to avoid these days when rescheduling. On "Reduced" days, you will only have ~50% the usual amount of reviews. On "Minimum" days, there will be ~0% reviews.

Reply:
(I don't know why "Insert a suggestion" or whatever it's called doesn't appear)